python/GafferUI/NumericWidget.py

```diff
@@ -141,6 +142,19 @@ def __keyPress( self, widget, event ) :
 
 		return False
 
+	def __wheel( self, widget, event ) :
+
+		assert( widget is self )
+
+		if not self.getEditable() :
+			return False
```

Comment:
That's a good idea, I added the check for focus and that `Ctrl` is pressed in https://github.com/GafferHQ/gaffer/pull/6045/commits/bf5886754d71dc70df8d281fd405575d8a0ffc84

Reply:
Thanks, the interaction does feel a lot more intentional with the focus and `Ctrl` checks in place.

One thing we'll still need to handle is the mouse wheel event falling through and scrolling the parent while you're attempting to adjust a value. This should be fairly apparent if you try using the scroll wheel to adjust a parameter on a shader like `standard_surface` or `principled_bsdf` while a few of the sections are expanded and the Node Editor's vertical scroll bar is visible. 

A solution for this might just be to stick a `qEvent.accept()` in: https://github.com/GafferHQ/gaffer/blob/cbd8e39a663cb9a17b3fdb15c59c2e848ba73d9a/python/GafferUI/Widget.py#L1271-L1281

Reply:
Thanks for pointing that out in `Widget`, I would have struggled a bit to locate where the signal was falling through. I don't understand why we need this for the wheel event but not others (except the foreign drag), but it works. And it doesn't break the other uses of `wheelSignal()` like viewport zooming.

Done in https://github.com/GafferHQ/gaffer/pull/6045/commits/4203049af5a41a9948e961aaccca3995f747d697. I assume based on other uses of `accept()` that we only want to accept if the signal handler returns `True`.

Reply:
Thanks, that's done the trick.